<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>420</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>calc</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 rgba(255, 4, 34, 255), stop:1 rgba(255, 28, 145, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="mPower">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string notr="true">X^n</string>
    </property>
   </widget>
   <widget class="QPushButton" name="n5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="n1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="n0">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>89</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0)</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="n3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mPrime">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Pr</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mRoot">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>âˆš</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dot">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="result">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>330</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="n9">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QLabel" name="screen">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string notr="true">2+2+5+48-8*94*/546542</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="del">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="n8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mMul">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mSub">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mFactorial">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="n6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>19</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="n7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bPerentheses">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zero">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="n4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mAdd">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mDiv">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>330</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="n2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ePerentheses">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
